lib/rake/extensiontask.rb

```diff
@@ -188,34 +217,12 @@ def define_compile_tasks(for_platform = nil, ruby_ver = RUBY_VERSION)
       # makefile depends of tmp_dir and config_script
       # tmp/extension_name/Makefile
       file "#{tmp_path}/Makefile" => [tmp_path, extconf, siteconf_path] do |t|
-        options = @config_options.dup
-
-        # include current directory
-        include_dirs = ['.'].concat(@config_includes).uniq.join(File::PATH_SEPARATOR)
-        cmd = [Gem.ruby, "-I#{include_dirs}", "-r#{File.basename(siteconf_path)}"]
-
-        # build a relative path to extconf script
-        abs_tmp_path = (Pathname.new(Dir.pwd) + tmp_path).realpath
-        abs_extconf = (Pathname.new(Dir.pwd) + extconf).realpath
-
-        # now add the extconf script
-        cmd << abs_extconf.relative_path_from(abs_tmp_path).to_s
-
-        # fake.rb will be present if we are cross compiling
-        if t.prerequisites.include?("#{tmp_path}/fake.rb") then
-          options.push(*cross_config_options(platf))
+        cross_platform = if t.prerequisites.include?("#{tmp_path}/fake.rb")
+          platf
         end
```

Comment:
Could you use the following style for readability?

```ruby
if ...
  cross_platform = platf
else
  cross_platform = nil
end
```

Or:

```ruby
cross_platform = nil
cross_platform = platf if ...
```

Reply:
Done!